### ISSUE TITLE
Mini selection issue

### ISSUE THREAD

**Issue by BlappoSpawn** [Bouncyrock/TaleSpire-Alpha-Public-Issue-Tracker#57]
**Describe the bug**
I walked behind a table and could not grab my mini because of minimal cover from a table

**To Reproduce**
1. walk behind table
2. have camera show only part of the base from isometric view
3. attempt to move
4. nothing

**Screenshots/Videos/Twitch Clips**
![screenshot 49](https://user-images.githubusercontent.com/46600569/51089059-ec32c800-1734-11e9-946f-f890ea2d21f4.png)

**Additional context**
obviously a minor issue because the workaround is turn the camera, but it feels odd when 90% of the mini is visible to not be able to select.

**Comment by ryvaeus** [comment]
I'm unable to replicate this even after building your exact setup with the same figurine model, and even with the base fully obscured by the table.

**Comment by BlappoSpawn** [comment]
I mentioned in features angle adjustment of camera,

Perhaps the Zoom or is different? or perhaps it is you an in GM mode? 
I will look at my scene again after work and attempt to find other factors

**Comment by ryvaeus** [comment]
>perhaps it is you an in GM mode?

Nope, I made sure I was in player mode when testing. Was trying as much camera finagling as possible but I just couldn't reproduce your issue. There may be other factors, or it could be unrelated to the camera at all.

**Comment by BlappoSpawn** [comment]
New one, behind the pillar, i dont know exactly what is causing it, because from 1 side of the pillar while behind it everything is normal... and then this side i can no longer move without rotating the camera

![screenshot 54](https://user-images.githubusercontent.com/46600569/51288791-beb37c00-19cb-11e9-99e8-a2b3a46f0284.png)

**Comment by cbaggers** [comment]
@BlappoSpawn @ryvaeus Thanks to both of you once again for all this great research across so many issues. The raycasts to check if we can pick up the creature are (most likely) too simplistic. I'll see what we can do to improve this. 

Tracking here: https://github.com/Bouncyrock/TaleSpire/issues/471

**Comment by cbaggers** [comment]
We made some improvements not long back to pickup actually. I'll rebuild these situations to see what places this issue still is affecting us
